test/suites/auth.sh

```diff
@@ -221,6 +221,36 @@ fine_grained_authorization() {
 
   # Check we can view the warning we just created.
   [ "$(lxc_remote query oidc:/1.0/warnings?recursion=1 | jq -r '[.[] | select(.last_message == "authorization warning")] | length')" = 1 ]
+
+  lxc auth group permission remove test-group server can_view_warnings
+
+  # Check we are not able to view any server config currently.
+  lxc config set user.foo bar
```

Comment:
Can we explicitly set and check access to core.trust_password and loki.auth.password are not accessible please?

Reply:
Can we also check that a non-admin cert (without fine grained auth) cant access these settings please.

Reply:
Yes sure. Do you think checking for one of them would be enough? They are all in the same `map[string]string`. Therefore I have added the `length == 0` checks to be sure you can't actually see anything.

Reply:
I'll add a follow up PR.

Reply:
Im just sort of paranoid about leaking passwords :)

Reply:
Fixed in https://github.com/canonical/lxd/pull/13502